docs/_articles/en/user-guide/metadata-operations.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Metadata Operations
+lang: en
+---
+
+## Source Diff
+The Source Diff command makes it easier to visualize the changes between files in your local project and the metadata in your org. You can look at diffs for an individual file or directory.
+
+### Usage
+Menu options `SFDX: Diff File Against Org` and `SFDX: Diff Folder Against Org` appear when you right-click a file or folder.  
+
+![Source Diff command](../../../images/source_diff.png)
+
+You can then view individual file diffs side-by-side. If a file doesn’t exist locally or in the org, it doesn't show up in the list.
+
+## Retrieve Source
+
+For non-source-tracked orgs, VS Code doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files unless you have conflict detection turned on. See [Detect Conflicts](./en/user-guide/detect-conflicts).
```

Comment:
There is no conflict detection on retrieve operations.  There is currently a suggestion in the docs to run the diff commands before retrieving.  

```suggestion
For non-source-tracked orgs, VS Code doesn’t automatically track changes to your org, so be sure to keep track of the changes you retrieve. Retrieving source from an org overwrites the local versions of the source files.
```